comms-templates/vulnerability-announcement-email.md

```diff
@@ -7,12 +7,17 @@ issue](vulnerability-announcement-issue.md)._
 
 TO: `kubernetes-announce@googlegroups.com, dev@kubernetes.io, kubernetes-security-announce@googlegroups.com, kubernetes-security-discuss@googlegroups.com, distributors-announce@kubernetes.io, kubernetes+announcements@discoursemail.com`
 
+Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted. Ask your fellow SRC members on how to get yourself added to these groups. 
```

Comment:
nit: can you format these a little differently to make them stand out from the other content a bit? Maybe quote them?
```suggestion
> Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted. Ask your fellow SRC members on how to get yourself added to these groups. 
```